intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -60,21 +60,21 @@ LLM Runtime assumes the same model format as [llama.cpp](https://github.com/gger
 git clone https://huggingface.co/meta-llama/Llama-2-7b-chat-hf
 
 # convert the pytorch model to ggml format
-python scripts/convert_model.py --outtype f32 --outfile ne-f32.bin model_path
+python scripts/convert.py --outtype f32 --outfile ne-f32.bin model_path
 
 # or convert the model without downloading it by hand (llama and llama2 are WIP) 
-python scripts/convert_model.py --outtype f32 --outfile EleutherAI/gpt-j-6b
+python scripts/convert.py --outtype f32 --outfile EleutherAI/gpt-j-6b
 
 # quantize weights of fp32 ggml bin
 # model_name: llama, llama2, mpt, falcon, gptj, starcoder, dolly
-# to neuarl engine graph optimized q4_j with 128 block_size format (recommended)
-python scripts/quant_bin.py --model_name llama2 --model_file ne-f32.bin --out_file ne-q4_j.bin --weight_dtype int4 --block_size 128 --compute_type int8
+# to neural engine graph optimized q4_j with 128 group_size format (recommended)
+python scripts/quantize.py --model_name llama2 --model_file ne-f32.bin --out_file ne-q4_j.bin --weight_dtype int4 --group_size 128 --compute_type int8
 
 # to ggml q4_0 format
-python scripts/quant_bin.py --model_name llama2 --model_file ne-f32.bin --out_file ne-q4_0.bin --weight_dtype int4
-# to neuarl engine graph optimized q4_j with 32 block_size format
+python scripts/quantize.py --model_name llama2 --model_file ne-f32.bin --out_file ne-q4_0.bin --weight_dtype int4
+# to neural engine graph optimized q4_j with 32 group_size format
```

Comment:
how about unifying to LLM runtime? seems we are still using neural engine ...

Reply:
or just make it simple: optimized INT4 model with group size 32

Reply:
done